services/listings/ormconfig.json

```diff
@@ -4,23 +4,14 @@
    "port": 5432,
    "database": "bloom",
    "synchronize": true,
-   "username": "martin",
    "logging": false,
    "entities": [
-      "dist/entity/**/*.ts",
       "src/entity/**/*.ts"
    ],
    "migrations": [
-      "dist/migration/**/*.ts",
       "src/migration/**/*.ts"
    ],
    "subscribers": [
-      "dist/subscriber/**/*.ts",
       "src/subscriber/**/*.ts"
-   ],
-   "cli": {
-      "entitiesDir": "src/entity",
-      "migrationsDir": "src/migration",
-      "subscribersDir": "src/subscriber"
-   }
-}
```

Comment:
I thought cli config is required?

Reply:
Actually I was not able to find 'cli' key anywhere in typeorm docs. Do you have any reference to manual with it?

Reply:
It's here: https://typeorm.io/#/using-cli